### PR TITLE
fix rendering issue related to react getting components mixed

### PR DESCRIPTION
**Pull request by KillariDev**

### CODE REVIEW

File: app/ts/components/App.tsx [hunk 1]
```diff
@@ -94,11 +136,13 @@ export function App() {
 					openRenewalContractIsApproved: currElement.approved === getOpenRenewalManagerAddress()
 				}
 			})
+			console.log('updated to', ensSubDomain)
```
Comment (jubalm) [inline app/ts/components/App.tsx]: this looks like a good place for `console.info`

File: app/ts/components/App.tsx [hunk 2]
```diff
@@ -25,6 +29,43 @@ const WalletComponent = ({ maybeAccountAddress }: WalletComponentProps) => {
...
+interface ErrorComponentProps {
+	show: boolean
+	message: string
+}
+const ErrorComponent = ({ show, message }: ErrorComponentProps) => {
+	if (show === false) return <></>
+	return <p class = 'error-component'> { message } </p>
+}
+
+interface ErrorComponentSignalProps {
```
Comment (jubalm) [inline app/ts/components/App.tsx]: functions pretty similar to `ErrorComponent`?

File: app/ts/components/requirements.tsx [hunk 3]
```diff
@@ -205,9 +233,28 @@ export const Create = ( { contentHashInput, resolutionAddressInput, loadingInfos
...
 	})
 
-	return <>
-		<div style = 'padding-top: 10px;'>
-			{ !immutable.value ? <div style = 'padding: 10px;'>
+	if (checkBoxes.deepValue === undefined) return <></>
+	const subDomain = checkBoxes.deepValue[checkBoxes.deepValue.length - 1]?.domainInfo.subDomain
+	if (subDomain === undefined) throw new Error('missing subdomain')
+
+	return <div style = 'padding-top: 10px;'>
```
Comment (jubalm) [inline app/ts/components/requirements.tsx:240]: ternaries multiple nested conditions tend to be a little more readable with `if` statements

**Review by jubalm — APPROVED** [review]

File: app/ts/components/requirements.tsx (hunk 3 above)
Comment (MicahZoltu) [reply app/ts/components/requirements.tsx:240]: My preference for giant complicated JSX like this is to use local components with something like
```tsx
const [ImmutableComponent] = useState(() => () => <>...</>)
```

File: app/ts/components/App.tsx (hunk 1 above)
Comment (KillariDev) [reply app/ts/components/App.tsx]: I'll just remove it, it was for debugging

**Pull request merged**